### PR TITLE
Ad react ropes issue1

### PR DESCRIPTION
**Pull request by apurvabakshi**

### CODE REVIEW

**Comment by apurvabakshi** [comment]
[React Issue 1: Add Footer](https://trello.com/c/yNTinvKm/1-react-issue-1-add-footer)

File: app/javascript/test/delete_this_too.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import assert from 'assert';
 
-describe('delete_me', () => {
+describe('test', () => {
   it('passes the tests but should be deleted when you start working on this', () => {
```
Comment (justinpearsonappfolio) [inline app/javascript/test/delete_this_too.js]: I think this file and the next one can be deleted -- they are just placeholders.

File: package.json [hunk 2]
```diff
@@ -33,6 +33,6 @@
   },
   "scripts": {
     "lint": "eslint app/javascript",
-    "test": "mocha --recursive --compilers js:babel-register --require app/javascript/testHelpers.js --require ignore-styles app/javascript/**/test"
+    "test": "mocha --recursive --compilers js:babel-register --require app/javascript/testHelpers.js --require ignore-styles app/javascript/image_sharing/test/components"
```
Comment (justinpearsonappfolio) [inline package.json]: Can you please move the `vendor/cache/` changes to their own commit? They clutter up this commit.

File: app/javascript/image_sharing/components/Header.js [hunk 3]
```diff
@@ -8,13 +8,13 @@ class Header extends Component {
   };
 
   render() {
-    const title = this.props.title;
+    const x = this.props.title;
```
Comment (justinpearsonappfolio) [inline app/javascript/image_sharing/components/Header.js]: Could you have this as `title` and not `x` for clarity? Sry, I think I did that to your code...

File: Gemfile [hunk 4]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.5.1'
+ruby '2.5.3'
```
Comment (justinpearsonappfolio) [inline Gemfile]: Can you move these house-keeping changes to a separate commit? They clutter up your real work.

**Pull request merged**